src/libos/src/net/io_multiplexing/poll.rs

```diff
@@ -19,6 +19,22 @@ bitflags! {
     }
 }
 
+impl PollEventFlags {
+    pub fn from_poll_status(poll_status: &crate::rcore_fs::vfs::PollStatus) -> Self {
```

Comment:
`PollEventFlags` is in effect deprecated (it is the old event polling mechanism). And the virtual files under /dev/ is not likely to be polled. So just remove the code related to `PollEventFlags` in DevFs.

Reply:
Done